your_daily_dose_malware/backends/virus_exchange.py

```diff
@@ -0,0 +1,85 @@
+import os
+import requests
+import time
+from pathlib import Path
+from datetime import datetime as dt
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from rich.progress import Progress
+
+class VirusExchangeScraper:
+    def __init__(self):
+        print("init")
+        self.driver = webdriver.Chrome()
+        self.login_url = "https://virus.exchange/users/log_in"
+        self.samples_url = "https://virus.exchange/samples"
+        self.wait = WebDriverWait(self.driver, 10)
+    
+    def login(self, email, password):
+        # Login to the Virus Exchange site
```

Comment:
Do they not have an API? Do we really need to login to download samples?
Are we able to maybe use request to send a post, to login, keep the cookie and then send it in another request to get the samples via get?

Reply:
The API didn't work

"Are we able to maybe use request to send a post, to login, keep the cookie and then send it in another request to get the samples via get?"

I tried it and it didn't work